src/components/MetricChart/colors.ts

```diff
@@ -0,0 +1,27 @@
+// Grafana classic palette
+export const colors = [
+    '#7EB26D', // 0: pale green
+    '#EAB839', // 1: mustard
+    '#6ED0E0', // 2: light blue
+    '#EF843C', // 3: orange
+    '#E24D42', // 4: red
+    '#1F78C1', // 5: ocean
+    '#BA43A9', // 6: purple
+    '#705DA0', // 7: violet
+    '#508642', // 8: dark green
+    '#CCA300', // 9: dark sand
+];
+
+export function colorHexToRGBA(htmlColor: string, opacity: number) {
```

Comment:
Maybe better to use special library for such manipulations. For example, https://www.npmjs.com/package/colord
With it, code will look like
```js
import {colord} from "colord";

export function colorToRgba(colorString: string; opacity: number) {
    const color = colord(colorString);
    if (!color.isValid()) {
        throw new Error(
            'Invalid color passed, the color should be in the html format. Example: #ff0033',
        );
    }
    
    return color.alpha(opacity).toRgbString();
}
```

Reply:
Good idea, thanks!